src/libraries/Microsoft.PowerFx.Core/Public/IDependencyFinder.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Core.Public
+{
+    /// <summary>
+    /// Exposes the ability parse a formula, walk the tree and return a list of dependencies
+    /// or variable names as strings.
+    /// </summary>
+    public interface IDependencyFinder
```

Comment:
This doesn't seem all that useful as a public interface. I see it's used for FormulaSet to avoid exposing DependencyFinder directly, but I'm not sure that makes sense.

Reply:
DependencyFinder's method is internal and static, couldn't you just use it directly without the reference?

Reply:
Made it internal